src/devices/machine/upd765.cpp

```diff
@@ -20,7 +20,14 @@
 #define LOG_LIVE    (1U << 12)  // Live states
 #define LOG_DONE    (1U << 13)  // Command done
 
-#define VERBOSE (LOG_GENERAL | LOG_WARN )
+//#define VERBOSE (LOG_GENERAL | LOG_WARN | LOG_COMMAND | LOG_DONE | LOG_STATE | LOG_RW | LOG_FIFO | LOG_TCIRQ)
+//#define VERBOSE (LOG_COMMAND)
+
+#ifdef _MSC_VER
+#define FUNCNAME __func__
+#else
+#define FUNCNAME __PRETTY_FUNCTION__
+#endif
```

Comment:
The rest of this file uses human-friendly log messages rather than function name macros.  Please follow that style rather than doing this.